### PR TITLE
Jupiter 10.18.0 adding vb plugin name

### PR DESCRIPTION
**Pull request by shpiNi**

### CODE REVIEW

File: configurations/plugins.template.ini [hunk 1]
```diff
@@ -104,4 +104,5 @@ ScheduledTask
 ScheduledTaskMetadata
 ScheduledTaskEventNotification
 ScheduledTaskContentDistribution
-FeedDropFolder
\ No newline at end of file
+FeedDropFolder
+Voicebase
```
Comment (shpiNi) [inline configurations/plugins.template.ini]: FYI @jessp01

File: plugins/integration/IntegrationProviderPlugin.php [hunk 2]
```diff
@@ -6,6 +6,8 @@
 {
 	const INTEGRATION_PLUGIN_NAME = 'integration';
 	
+	//exteding classes should implement this function to make sure
```
Comment (shpiNi) [inline plugins/integration/IntegrationProviderPlugin.php]: FYI @hilak

File: plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php [hunk 3]
```diff
@@ -71,6 +71,12 @@ protected function doDispatch(KalturaBatchJob $job, KalturaIntegrationJobData &$
 	
 	protected function doClose(KalturaBatchJob $job, KalturaIntegrationJobData &$data, KalturaVoicebaseJobProviderData $providerData)
 	{
+		$entryId = $providerData->entryId;
+		KalturaLog::debug("checking entry id - $entryId");
+		$this->clientHelper = VoicebasePlugin::getClientHelper($providerData->apiKey, $providerData->apiPassword);
+		if($this->clientHelper->checkExitingExternalContent($entryId))
```
Comment (hilak) [inline plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php]: @shpiNi  I believe that this is a typo - probably meant to be checkExistingExternalContent, please change here and on the VoiceBaseClientHelper class

File: plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php [hunk 4]
```diff
@@ -71,6 +71,12 @@ protected function doDispatch(KalturaBatchJob $job, KalturaIntegrationJobData &$
 	
 	protected function doClose(KalturaBatchJob $job, KalturaIntegrationJobData &$data, KalturaVoicebaseJobProviderData $providerData)
 	{
+		$entryId = $providerData->entryId;
+		KalturaLog::debug("checking entry id - $entryId");
+		$this->clientHelper = VoicebasePlugin::getClientHelper($providerData->apiKey, $providerData->apiPassword);
```
Comment (hilak) [inline plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php]: @shpiNi  I don't think there is really a point in this static function. The VoiceBaseClientHelper is not a singleton. You only instantiate it in 2 places in the code. Just call the class constructor.(Aesthetic comment, not urgent for Tue deployment)

File: plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php (hunk 3 above)
Comment (shpiNi) [inline plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php]: @hilak done.

File: plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php (hunk 4 above)
Comment (shpiNi) [inline plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php]: @hilak I thought it's more elegant getting it through the plugin, but basically I agree. I'll handle it if i have another round of code fixes here.

**Pull request merged**